pkg/controller/switch_lb_rule_test.go

```diff
@@ -4,6 +4,11 @@ import (
 	"reflect"
 	"testing"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
This package import should be placed together with `corev1 "k8s.io/api/core/v1"`.